MIGRATION.md

```diff
@@ -0,0 +1,86 @@
+0.19 to 0.21
+============
+
+## Configuration
+
+The biggest changes from 0.19 to 0.21 are related to the application.conf file, which has been restructured significantly.
+
+The configuration for backends now is all contained within a `backend` stanza, which specifies 1 stanza per backend and a default backend., as follows:
```

Comment:
`per name per backend`? Since it's in theory possible to have two different JES backends using slightly different authentications?